listenbrainz/webserver/templates/user/missing_data.html

```diff
@@ -0,0 +1,25 @@
+{% extends 'base-react.html' %}
+
+{% block title %}Missing MusicBrainz Data of "{{ user.musicbrainz_id }}" - ListenBrainz{% endblock %}
+
+{% block content %}
+  <h2 class="page-title">Missing MusicBrainz Data of {{ user.musicbrainz_id }}</h2>
+  {% if current_user.is_authenticated and current_user == user %}
+
+    <h3>Directly export missing MusicBrainz data</h3>
+      <p>Share a hand to the community effort of gathering and marking all known music in existence! Such small submissions help
+         us grow more :D.
+      </p>
+      <div class="row">
+      <div class="well col-sm-8 col-sm-offset-2 col-md-6 col-md-offset-3">
+        <div id="react-container">
+        </div>
+      </div>
+      </div>
```

Comment:
Getting rid of the first div (that has row class), makes the footer glitch. The second div (that has well) makes the page non responsive, but I have removed the well class. Hope that works. Hence I _have not removed any div_.

(If there are any other methods to get the same effect, while making the codebase better, please do let me know, since I am not as adept in frontend as my backend, so feel free to drop advice and to be pedantic, as it will help me get better.)